infrastructure/k8s-config/clusters/kit-infrastructure/kit-provisioner.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata: 
+  name: kit
+spec:
+  amiFamily: AL2
+  subnetSelector:
+    kit.sh/stack: KITInfrastructure
+    aws-cdk:subnet-type: Private
+  securityGroupSelector:
+    kit.sh/stack: KITInfrastructure
+---
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata: 
+  name: kit
```

Comment:
Why do we need another provisioner? Can't we extend the existing [default provisioner](https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/infrastructure/k8s-config/clusters/kit-infrastructure/provisioner.yaml)?

Having a KIT provisioner, does it mean we will need to update the pods for guest clusters which rely today on default provisioner to provision nodes?

Reply:
Had an offline discussion, we don't need to update guest cluster pods to provision nodes using KIT provisioner with the latest Karpenter version.

Reply:
Then what is the purpose of this provisioner being here ?